_sass/custom/custom.scss

```diff
@@ -226,14 +334,13 @@ p.label {
 // Modifies margins in xl layout to support TOC
 .side-bar {
   @include mq(xl) {
-    width: calc((100% - #{$nav-width + $content-width + $toc-width}) / 2 + #{$nav-width});
-    min-width: $nav-width;
+    width: calc(#{$nav-width} + 1rem);
   }
 }
 
 .main {
   @include mq(xl) {
-    margin-left: calc((100% - #{$nav-width + $content-width + $toc-width}) / 2 + #{$nav-width});
+    max-width: calc(100% - (#{$nav-width + $toc-width} + 4rem));
   }
 }
```

Comment:
These changes make the page consume the entire width of the window which neither the main website, nor the header is aligned with.

Reply:
The requirement was to give users more space for the main content. However, I added a maximum width for the main content in[ this PR](https://github.com/opensearch-project/documentation-website/pull/2259).